BMM.Core/ViewModels/Base/DocumentsViewModel.cs

```diff
@@ -106,25 +112,25 @@ protected override async Task DocumentAction(Document item)
         public DocumentsViewModel(IDocumentFilter documentFilter = null)
         {
             Filter = documentFilter ?? new NullFilter();
-            Documents = new BmmObservableCollection<Document>();
+            Documents = new BmmObservableCollection<IDocumentPO>();
 
             CurrentTrack = Mvx.IoCProvider.Resolve<IMediaPlayer>().CurrentTrack;
             _currentTrackChangedToken = Messenger.Subscribe<CurrentTrackChangedMessage>(message =>
             {
                 CurrentTrack = message.CurrentTrack;
+                RefreshTracksStatesInDocuments();
             });
 
             ConnectionStatus = Mvx.IoCProvider.Resolve<IConnection>().GetStatus();
             _connectionStatusChangedToken = Messenger.Subscribe<ConnectionStatusChangedMessage>(message =>
             {
                 ConnectionStatus = message.ConnectionStatus;
```

Comment:
this should also RefreshTracksStatesInDocuments() since it's dependent on the connection status